mne/io/brainvision/tests/test_brainvision.py

```diff
@@ -453,4 +484,21 @@ def test_events():
     assert_equal(raw.info['chs'][-1]['ch_name'], 'STI 014')
 
 
+def test_brainvision_with_montage():
+    """Test reading embedded montage information"""
+    raw = read_raw_brainvision(vhdr_v2_path, eog=eog, misc=['ReRef'])
+    for i, d in enumerate(raw.info['dig'], 1):
+        assert_equal(d['coord_frame'], FIFF.FIFFV_COORD_HEAD)
+        assert_equal(d['ident'], i)
+        assert_equal(d['kind'], FIFF.FIFFV_POINT_EEG)
+        assert_equal(len(d['r']), 3)
+
+    raw_none = read_raw_brainvision(vhdr_v2_path)
+    for r, n in zip(raw.info['chs'], raw_none.info['chs']):
+        try:
```

Comment:
why do you need to try here?

Reply:
Please avoid the try to failure is seen on channels for which this is expected

Reply:
Okay, I changed to if instead of try/except. Hope this is what you meant.